### ISSUE TITLE
install requirements_test.txt on dev boxes

### ISSUE THREAD

**Issue by binary1230** [magfest/ubersystem-puppet#154]
Currently, to run the unit tests directly without running through tox.ini (through pycharm, for instance), we need to run ```pip install -r requirements_test.txt```

What we should do is check if dev_box is set to true in the puppet config, and then add a line that runs ```pip install -r requirements_test.txt``` in https://github.com/magfest/ubersystem-puppet/blob/master/manifests/install_dependencies.pp#L6

**Comment by RobRuana** [comment]
Was this fixed by: https://github.com/magfest/simple-rams-deploy/pull/30?

**Comment by binary1230** [comment]
nah, seperate issue.

I can run the tests from pycharm now (with a little weirdness but correct result), however, I had to do a workaround of manually installing the stuff in requirements_test.txt.  Puppet should do that if dev_box is true.

I'm getting some other odd issues running unit tests locally under vagrant+windows related to sqllite, will post those later.

**Comment by RobRuana** [comment]
Yeah, we've really grown beyond the capabilities of SQLite. Our tests are just barely holding together running against SQLite, which is why I opened this issue: https://github.com/magfest/ubersystem/issues/2517
